README.md

```diff
@@ -30,206 +38,117 @@ Run Lightning Terminal with a local `lnd` instance:
-| **v0.10.2-alpha** | v0.16.0-beta |
-| **v0.10.1-alpha** | v0.16.0-beta |
-| **v0.10.0-alpha** | v0.16.0-beta |
-| **v0.9.2-alpha**  | v0.16.0-beta |
-| **v0.9.1-alpha**  | v0.16.0-beta |
-| **v0.9.0-alpha**  | v0.16.0-beta |
-| **v0.8.6-alpha**  | v0.15.4-beta |
-| **v0.8.5-alpha**  | v0.15.4-beta |
-| **v0.8.4-alpha**  | v0.15.1-beta |
-| **v0.8.3-alpha**  | v0.15.1-beta |
-| **v0.8.2-alpha**  | v0.15.1-beta |
-| **v0.8.1-alpha**  | v0.15.1-beta |
-| **v0.8.0-alpha**  | v0.15.1-beta |
-| **v0.7.1-alpha**  | v0.14.3-beta |
-| **v0.7.0-alpha**  | v0.14.3-beta |
-| **v0.6.7-alpha**  | v0.13.3-beta |
-| **v0.6.6-alpha**  | v0.13.3-beta |
-| **v0.6.5-alpha**  | v0.13.3-beta |
-| **v0.6.4-alpha**  | v0.13.3-beta |
-| **v0.6.3-alpha**  | v0.13.3-beta |
-| **v0.6.2-alpha**  | v0.13.3-beta |
-| **v0.6.1-alpha**  | v0.13.3-beta |
-| **v0.6.0-alpha**  | v0.13.3-beta |
-| **v0.5.2-alpha**  | v0.12.0-beta |
-| **v0.5.1-alpha**  | v0.12.0-beta |
-| **v0.5.0-alpha**  | v0.12.0-beta |
-| **v0.4.1-alpha**  | v0.11.1-beta |
-| **v0.4.0-alpha**  | v0.11.1-beta |
-| **v0.3.4-alpha**  | v0.11.1-beta |
-| **v0.3.3-alpha**  | v0.11.1-beta |
-| **v0.3.2-alpha**  | v0.11.1-beta |
-| **v0.3.1-alpha**  | v0.11.1-beta |
-| **v0.3.0-alpha**  | v0.11.1-beta |
-| **v0.2.0-alpha**  | v0.11.0-beta |
-
-LiT offers two main operating modes, one in which [`lnd` is running inside the
-LiT process (called "lnd integrated mode", set by `lnd-mode=integrated` config
-option)](docs/config-lnd-integrated.md) and one in which [`lnd` is running in
-a standalone process on the same or remote machine (called "lnd remote mode",
-set by `lnd-mode=remote` config option)](docs/config-lnd-remote.md).
-
-In addition to those main modes, the individual bundled daemons (Faraday, Loop
-and Pool) can be toggled to be integrated or remote as well, or as disabled.
-This offers a large number of possible configuration combinations, of which not
-all are fully supported due to technical reasons.
-
-The following table shows the supported combinations:
+
+If you’d prefer to compile from source code please
+follow [these instructions](./docs/compile.md).
+
+## Component Compatibility
+
+This section describes version and mode compatibility across Lightning Terminal
+(LiT) and its component daemons.
+
+### LND Version Compatibility
+
+LiT can operate in two modes: [integrated](docs/config-lnd-integrated.md) and
+[remote](docs/config-lnd-remote.md). Version compatibility between LiT and `lnd`
+matters only when running in remote mode. The lnd version bundled in
+integrated mode is always compatible.
+
+Each LiT release should specify the minimum compatible `lnd` version in its 
+[GitHub release
+notes](https://github.com/lightninglabs/lightning-terminal/releases). If a
+release does not document compatibility, the required information is available
+in the archived compatibility table in
+[docs/compatibility.md](./docs/compatibility.md).
+
+### Other Daemon Compatibility
```

Comment:
Nit: Would potentially rename this to something like: "Compatibility With Additional Daemons"

But I don't have a strong preference, so feel free to keep this as is if you think this current title sounds natural.